dpc/src/record/payload.rs

```diff
@@ -18,19 +18,21 @@ use snarkvm_utilities::{FromBytes, ToBytes};
 
 use std::io::{Read, Result as IoResult, Write};
 
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Default)]
-pub struct Payload([u8; 32]);
+pub const PAYLOAD_SIZE: usize = 128;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Payload([u8; PAYLOAD_SIZE]);
 
 impl Payload {
     pub fn to_bytes(&self) -> &[u8] {
         &self.0[..]
     }
 
     pub fn from_bytes(bytes: &[u8]) -> Self {
-        assert_eq!(bytes.len(), 32);
+        assert_eq!(bytes.len(), PAYLOAD_SIZE);
```

Comment:
this assertion is unnecessary, as `copy_from_slice` performs this check internally; the `payload` binding below could then be
```
let mut payload = [0u8; PAYLOAD_SIZE];
```